Cargo.toml

```diff
@@ -25,3 +25,4 @@ proc-macro = true
 async-trait = "0.1.50"
 futures = "0.3.16"
 tokio = { version = "1.16.1", features = ["sync"] }
+macrotest = "1.0.12"
```

Comment:
Could you please also add instructions to the README on how to bless the tests? For example by adding a `## Development` section after the `## Features` section.